src/main/java/org/opensearch/flowframework/common/CommonValue.java

```diff
@@ -33,6 +33,14 @@ private CommonValue() {}
     public static final String WORKFLOW_STATE_INDEX_MAPPING = "mappings/workflow-state.json";
     /** Workflow State index mapping version */
     public static final Integer WORKFLOW_STATE_INDEX_VERSION = 1;
+    /** Master Key Index Name */
+    public static final String MASTER_KEY_INDEX = ".plugins-master-key";
```

Comment:
we should include an indication of our plugin name in all indices, might mean changing our other indices names too

Reply:
Not sure if its necessary, for example, ML Commons config system index is just `.plugins-ml-config`. But I can change this to `.plugins-flow-framework-config`

Reply:
yeah meant just some indication like `ml` since `.plugin-config` could sound like for all plugins. these are other system indices I found, we can do just `.plugins-flow-config` if we want to make it short, either is fine?
```
".plugins-ml-config, .plugins-ml-connector, .plugins-ml-model-group, .plugins-ml-model, "
        + ".plugins-ml-task, .plugins-ml-conversation-meta, .plugins-ml-conversation-interactions, .opendistro-alerting-config, .opendistro-alerting-alert*, "
        + ".opendistro-anomaly-results*, .opendistro-anomaly-detector*, .opendistro-anomaly-checkpoints, .opendistro-anomaly-detection-state, "
        + ".opendistro-reports-*, .opensearch-notifications-*, .opensearch-notebooks, .opensearch-observability, .ql-datasources, "
        + ".opendistro-asynchronous-search-response*, .replication-metadata-store, .opensearch-knn-models, .geospatial-ip2geo-data*
 ```

Reply:
addressed in most recent commit